### ISSUE TITLE
🌋 E|F ➾ plugin-contract-types : export Storage type that it can be used on the frontend app

### ISSUE THREAD

**Issue by zamrokk** [ecadlabs/taqueria#1291]
### 🚥 Status

- [X] 🔮 Product/Feature Design
- [x] ✍🏻 Architecture/Implementation Planning
- [x] 🏗️ Under Construction
- [x] ⛳ Feature Complete
- [ ] ⛵ GTM / Release

### ✅ To Do

I have this generated code 

```
import { ContractAbstractionFromContractType, WalletContractAbstractionFromContractType } from './type-utils';
import { address, MMap, nat, ticket } from './type-aliases';

type Storage = {
    feedback: string;
    pokeTraces: MMap<address, {
        feedback: string;
        receiver: address;
    }>;
    ticketOwnership: MMap<address, ticket>;
};

type Methods = {
    init: (
        _0: address,
        _1: nat,
    ) => Promise<void>;
    poke: () => Promise<void>;
    pokeAndGetFeedback: (param: address) => Promise<void>;
};

type MethodsObject = {
    init: (params: {
        0: address,
        1: nat,
    }) => Promise<void>;
    poke: () => Promise<void>;
    pokeAndGetFeedback: (param: address) => Promise<void>;
};

type contractTypes = { methods: Methods, methodsObject: MethodsObject, storage: Storage, code: { __type: 'PokeGameCode', protocol: string, code: object[] } };
export type PokeGameContractType = ContractAbstractionFromContractType<contractTypes>;
export type PokeGameWalletType = WalletContractAbstractionFromContractType<contractTypes>;

```

I want Storage type to be exported too, thus I can use it to refers on different fetched storage I have on my React app in typescript

### 📐 Feature Description

Declare Storage type 

### 🔭 Scope

🏎️ In Scope: 
- ... 
 
 🙅‍♂️ Out of Scope: 
- ... 
 
 

### ⚖️ Acceptance Criteria

✨ Features: 
- [ ] ... 
 
 📚 Docs: 
- [ ] ... 

### 🦄 Sizing

0.5

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hu3man** [comment]
I can see this being quite useful, thanks @zamrokk!

@ricklove @jchenche, any ideas on the complexity of this?

**Comment by zamrokk** [comment]
I think is is super easy, just add "export" keyword before Storage definition

`export type Storage = {`

this was my workaround

**Comment by michaelkernaghan** [comment]
export type Storage = {
  feedback: string;
  pokeTraces: MMap<
    address,
    {
      feedback: string;
      receiver: address;
    }
  >;
};

**Comment by hu3man** [comment]
@AlirezaHaghshenas is looking into adding this fix into the current release

**Comment by AlirezaHaghshenas** [comment]
closed by #1483
